lib/search_it.php

```diff
@@ -2000,9 +2000,13 @@ public function deleteCache($_indexIds = false)
         if ($_indexIds === false) {
             // delete entire search-cache
             $delete = rex_sql::factory();
-            $delete->setQuery('TRUNCATE '. self::getTempTablePrefix() . 'search_it_cacheindex_ids');
-            $delete->setQuery('TRUNCATE '. self::getTempTablePrefix() . 'search_it_cache');
-
+            if ($delete->inTransaction()) {
```

Comment:
Ich habe gerade mal nachgeschaut, die Methode wurde in Rex 5.6 eingeführt. Daher müsstest du die Mindestversion in der package.yml von 5.5 auf 5.6 anheben.

Reply:
wird erledigt.